proxy/httpproxy/rawkv.go

```diff
@@ -49,7 +50,13 @@ type RawResponse struct {
 }
 
 func (h rawkvHandler) New(ctx context.Context, r *RawRequest) (*RawResponse, int, error) {
-	id, err := h.p.New(ctx, r.PDAddrs, config.Default())
+	var id proxy.UUID
+	var err error
+	if h.c == nil {
```

Comment:
It seems it will never be nil.

Reply:
I think this check can make the code safe, although h.c cannot be nil for current usage of rawkvHandler.

Reply:
em, how about add a `getConfig()` method to `rawkvHandler`, inside the method, when c is nil, return the default config. In this way, you will not have to pay extra attention in `newHTTPProxyHandler`.